adapters/kueezrtb/params_test.go

```diff
@@ -0,0 +1,42 @@
+package kueezrtb
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/prebid/prebid-server/v3/openrtb_ext"
+)
+
+func TestValidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json schema. %v", err)
+	}
+
+	for _, validParam := range validParams {
+		if err := validator.Validate(openrtb_ext.BidderKueezRTB, json.RawMessage(validParam)); err != nil {
+			t.Errorf("Schema rejected valid params: %s", validParam)
+		}
+	}
+}
+
+func TestInvalidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json schema. %v", err)
+	}
+
+	for _, p := range invalidParams {
+		if err := validator.Validate(openrtb_ext.BidderKueezRTB, json.RawMessage(p)); err == nil {
+			t.Errorf("Schema allowed invalid params: %s", p)
+		}
+	}
+}
+
+var validParams = []string{
+	`{"cId": "provided_cid_123"}`,
+}
+
+var invalidParams = []string{
+	`{"cId": 3898334}`,
```

Comment:
what happend if You add:
var invalidParams = []string{
	`{}`,
	`{"cId": 123}`,
	`{"cId": true}`,
	`{"cId": null}`,
	`{"cId": ""}`,
	`{"cId": "   "}`,
	`{"cId": "cid_with_invalid_chars!@#"}`,
	`{"cId": "valid_cid", "extra_field": "unexpected"}`,
	`{"cId": "a_very_long_connection_id_exceeding_the_maximum_allowed_length_..."}`,
	`[]`,
	`["cId"]`,
	`"string_value"`,
	`42`,
	`true`,
	`{"cId": "valid_cid"`,
	`{"cId":}`,
	`{"cid": "valid_cid"}`,
}
please add more invalidParams

Reply:
Added more..